configure-places-in-the-sdk/places-extension/using-places-without-active-region-monitoring.md

```diff
@@ -0,0 +1,415 @@
+# Using Places Without Active Region Monitoring
+
+Use cases for your application may not require active region monitoring. Adobe Places can still be used to get your users' location data integrated with other Experience Platform products.
+This document will show how to do a Point of Interest (POI) membership check only at the time of collecting the user's location (latitude and longitude). The document assumes the developer will collect the device's location using the APIs provided by the target platform's operating system.
```

Comment:
- Please add a line space between the 1st sentence and this section (starts with "This document...).
- Please reword the beginning of the 2nd sentence to read:
"This section explains how to complete a POI membership check only at the time of collecting the user's location (latitude and longitude). 

**Prerequisite**: 
The developer will collect the device's location using the APIs provided by the target platform's operating system.